tests/test_scheduler.py

```diff
@@ -82,15 +83,19 @@ def test_one_cycle_scheduler_not_step_with_optimizer_single_process(self):
         debug_launcher(partial(one_cycle_test, num_processes=1, step_scheduler_with_optimizer=False), num_processes=1)
 
     def test_lambda_scheduler_steps_with_optimizer_multiprocess(self):
+        AcceleratorState._reset_state(True)
```

Comment:
Since it switches from single cpu to multi-CPU within the same process/time period, the state needs to be reset